<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HordMethodForm</class>
 <widget class="QWidget" name="HordMethodForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Метод хорд</string>
  </property>
  <widget class="QLabel" name="label_a">
   <property name="text">
    <string>a:</string>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="field_a">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="text">
    <string>b:</string>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="field_b">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tolerance">
   <property name="text">
    <string>Допуск:</string>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="field_tolerance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="text">
    <string>Вычислить</string>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
